uint16 pgain
---
bool success
